report/report-generators/src/main/kotlin/com/testerum/report_generators/dirs/ReportDirs.kt

```diff
@@ -10,7 +10,7 @@ import java.time.format.DateTimeFormatter
 import kotlin.streams.asSequence
 import java.nio.file.Path as JavaPath
 
-object RunnerDirs {
+object ReportDirs {
```

Comment:
since we're here, let's also remove the following unused methods: ``getBasicStepsDir``, ``getSeleniumDriversDir``